networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/AttestationTopicHandler.java

```diff
@@ -33,25 +34,32 @@
 public class AttestationTopicHandler extends GossipTopicHandler<Attestation> {
 
   private static final Logger LOG = LogManager.getLogger();
-  private static final Topic ATTESTATIONS_TOPIC = new Topic("/eth2/beacon_attestation/ssz");
```

Comment:
Just to confirm - we're intentionally dropping the old interop topic?  Should we keep it around until its actually removed from the spec?

Reply:
Good question. Supporting both modes would require a good amount of effort tho. So I asked to the EF guys and will take action accordingly.

Reply:
After some offline discussion, moving forward with not having the interop topic for now, and implementing if deemed necessary later on.